content/50_workshop_5_accelerating_sdlc/40_progressive_delivery/03_canary_rollback.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Automated Canary Rollback"
+chapter = false
+weight = 403
++++
+
+Now we will generate some failed requests to trigger an automated rollback.
+
+During the canary analysis you can generate HTTP 500 errors and high latency to test if Flagger pauses and rolls back the faulted version.
+
+Trigger another canary release:
+
+```sh
+cat << EOF | tee overlays/podinfo.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podinfo
+  namespace: demo
+spec:
+  template:
+    spec:
+      containers:
+        - name: podinfod
+          image: stefanprodan/podinfo:3.1.2
+          env:
+            - name: PODINFO_UI_LOGO
+              value: https://eks.handson.flagger.dev/cuddle_bunny.gif
+EOF
+```
+
+Apply changes:
+
+```sh
+git add -A && \
+git commit -m "update podinfo" && \
+git push origin master && \
+fluxctl sync --k8s-fwd-ns flux
+```
+
+View Flagger logs with:
+
+```sh
+kubectl -n appmesh-system logs deployment/flagger -f | jq .msg
+```
+
+Exec into the tester pod and generate HTTP 500 errors:
+
+```sh
+kubectl -n demo exec -it $(kubectl -n demo get pods -o name | grep -m1 flagger-loadtester | cut -d'/' -f 2) bash
```

Comment:
Do we need to split this section into 2 code blocks? One to exec and then another one for the hey commands? The hey commangs could be `&&` together?